tests/console/velociraptor_client_event_collect.pm

```diff
@@ -49,53 +45,52 @@ sub run {
         }
     }
 
-    # create config dir to run server container
-    assert_script_run "cd /root";
-    assert_script_run "mkdir -p vrr/config vrr/data vrr/logs vrr/definitions";
-
-    # install podman and required dependencies
-    zypper_call "in cni podman";
-
-    # run server as container
-    assert_script_run "podman run -d --rm     --name vrr     -v /root/vrr/data:/data:Z     -v /root/vrr/config:/config:Z     -v /root/vrr/logs:/logs:Z     -v /root/vrr/definitions:/definitions:Z     -p 8001:8001     -p 8889:8889     -p 8000:8000     -p 8003:8003     registry.opensuse.org/security/sensor/containers/linux-security-sensor";
-
-    # verify container running
-    validate_script_output("podman ps", qr/vrr/);
-    validate_script_output("podman container inspect --format='{{.State.Running}}' vrr", qr/true/);
 
     # velociraptor client install
     zypper_call "in velociraptor-client";
-
     systemctl "enable velociraptor-client";
     systemctl "is-enabled velociraptor-client";
 
+    # generate config files
+    assert_script_run "velociraptor-client config generate > server.conf";
+    script_output 'yq -i ".defaults.event_max_wait = 1" server.conf';
+    script_output 'yq -i ".defaults.event_max_wait_jitter = 1" server.conf';
+    script_output 'yq -i ".defaults.event_change_notify_all_clients = true" server.conf';
+    background_script_run "velociraptor-client frontend -v --config server.conf > /dev/null 2>&1 &";
+    assert_script_run "velociraptor-client config client --config server.conf > client.conf";
+
+    # update config file
+    script_output 'yq -i ".Client.max_poll = 1" client.conf';
+    script_output 'yq -i ".Client.max_poll_std = 1" client.conf';
+    script_output 'yq -i ".Client.min_poll = 1" client.conf';
+    script_output 'yq -i ".Client.default_max_wait = 1" client.conf';
```

Comment:
Are these `yq` modifications to the config file? or is it something else? - otherwise maybe its better to use a pre-generated config file?

Reply:
the config file generated by velociraptor client and after i only adjust values specific to this test so it wont take long to collect events.